### PR TITLE
Add details assignment

### PR DESCRIPTION
**Pull request by IlmaMannix**
image

### CODE REVIEW

**Review by AldiNFitrah — APPROVED** [review]
LGTM, thankss!

File: src/resources/app/pages/course/Assignment.vue [hunk 1]
```diff
@@ -12,21 +12,33 @@
...
+      class="mt-5"
     >
       <a :href="this.assignment.linkUrl" target="_blank" class="mt-2 mb-10">
         {{ this.assignment.linkUrl }}
       </a>
     </div>
+    <div id="detail-assignment" class="mt-5">
+      <v-row>
+        <v-col cols="2"> Status </v-col>
+        <v-col> :No Submission </v-col>
```
Comment (Zsaschz) [inline src/resources/app/pages/course/Assignment.vue]: add space maybe? ` : No Submission`

File: src/resources/app/pages/course/Assignment.vue [hunk 2]
```diff
@@ -12,21 +12,33 @@
...
+      </v-row>
+      <v-row>
+        <v-col cols="2"> Due Date </v-col>
+        <v-col> :{{ date(this.assignment.dueDate) }} </v-col>
```
Comment (Zsaschz) [inline src/resources/app/pages/course/Assignment.vue]: this too

File: src/resources/app/pages/course/Assignment.vue [hunk 3]
```diff
@@ -12,21 +12,33 @@
...
+      </v-row>
+      <v-row v-if="this.assignment.point">
+        <v-col cols="2"> Max Point </v-col>
+        <v-col> :{{ this.assignment.point }} </v-col>
```
Comment (Zsaschz) [inline src/resources/app/pages/course/Assignment.vue]: and this

**Review by Zsaschz — APPROVED** [review]
LGTM!

**Pull request merged**